python/.clang-format

```diff
@@ -0,0 +1 @@
+DisableFormat: true
```

Comment:
Instead of turning `clang-format` off for everything, let's just disable it for the internals of each of the `Wrap` methods.